packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -158,17 +158,20 @@ class UnreferencedStateTracker {
     /** Logs an error with the given properties if the node  is inactive. */
     public logIfInactive(
         logger: ITelemetryLogger,
-        eventName: string,
+        eventSuffix: "Changed" | "Loaded" | "Revived",
         currentTimestampMs: number,
         deleteTimeoutMs: number,
         inactiveNodeId: string,
+        viaHandle?: boolean,
```

Comment:
There are other modes, like external request, internal request - would be nice to capture them all through enum

Reply:
Added enums for these.